Sources/MapboxCoreNavigation/AlternativeRoute.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+import MapboxDirections
+import MapboxNavigationNative
+import Turf
+
+/**
+ Representation of an alternative route with relation to the original.
+ 
+ This object contains main and alternative routes which are sharing same origin and destination points, but differ at some point.
+ */
+public class AlternativeRoute: Identifiable {
+    /// Alternative route identificator type
+    public typealias ID = UInt32
+    /// Breif stats of a route for travelling
+    public typealias Stats = (distance: LocationDistance, duration: TimeInterval)
+
+    /// Alternative route identificator.
+    ///
+    /// It is unique withing the same navigation session.
+    public let id: ID
+    /// Original (main) route data
+    public let indexedRouteResponse: IndexedRouteResponse
+    /// Intersection on the main route, where alternative route branches.
+    public let mainRouteIntersection: Intersection
+    /// Intersection on the alternative route, where it splits from the main route.
+    public let alternativeRouteIntersection: Intersection
+    /// Alternative route statistics, counting from the split point.
+    public let statsFromFork: Stats
```

Comment:
“Fork” could be confused with `ManeuverType.reachFork`. “Split point” isn’t a very intuitive term either; it sounds too much like a lane split (which happens at a fork in the road). I think the point at which two routes begin to take different paths is usually called the “divergence” or “deviation” point.